pkgs/tools/nix/info/info.sh

```diff
@@ -69,8 +69,14 @@ nixev() {
     nix-instantiate --eval --strict -E "$1"
 }
 
+# use `nix eval` command
+nixev2() {
+    nix eval --raw "$1"
+}
+
 desc_system() {
-    nixev '(import <nixpkgs> {}).stdenv.hostPlatform.system'
+    nixev '(import <nixpkgs> {}).stdenv.hostPlatform.system' 2>/dev/null ||
+        nixev2 'nixpkgs#stdenv.hostPlatform.system'
```

Comment:
Entry for nixpkgs is not guaranteed to be in the flake registry either. (Some overrides the registry to custom values or disables the registry completely)

Reply:
What does the nixpkgs here do exactly, isn't this just `builtins.currentSystem`?

Reply:
Oh yes,`builtins.currentSystem` seems to do exactly what we want and works everywhere. I have no idea why originally nixpkgs is used. Changed.